lesson_02/main.py

```diff
@@ -34,18 +26,23 @@ def fetch_sales():
         shutil.rmtree(raw_dir)
     os.makedirs(raw_dir, exist_ok=True)
 
-    response = requests.get(
-        url='https://fake-api-vycpfa6oca-uc.a.run.app/sales',
-        params={'date': date, 'page': 1},
-        headers={'Authorization': AUTH_TOKEN},
-    )
+    page = 1
+    while True:
+        response = requests.get(
+            url='https://fake-api-vycpfa6oca-uc.a.run.app/sales',
+            params={'date': date, 'page': page},
+            headers={'Authorization': AUTH_TOKEN},
+        )
+
+        if response.status_code == 200:
+            with open(os.path.join(raw_dir, f'sales_{date}_page_{page}.json'), 'w') as file:
```

Comment:
not right name of json file. 
<img width="850" alt="Screenshot 2024-10-02 at 10 18 38" src="https://github.com/user-attachments/assets/2ef66bb1-f954-48f6-b521-a4d91131ba3a">